src/plugins/wizard/public/application/components/workspace.scss

```diff
@@ -16,9 +16,9 @@
   }
 
   &__handFieldSvg {
-    animation: wizDragAnimation 2s ease-in-out infinite alternate;
+    animation: wizDragAnimation 6s ease-in-out infinite forwards;
```

Comment:
do we have the specs for this are we kind of eyeballing it?

Reply:
For now its eyeballing. Dont want it to be too distracting, which is why i changed it to include a pause and take longer to loop.